org.gnome.HexGL.json

```diff
@@ -61,7 +48,7 @@
                 {
                     "type" : "git",
                     "url" : "https://github.com/ebassi/graphene",
-                    "tag" : "1.10.0"
+                    "tag" : "1.10.8"
```

Comment:
Graphene is in the GNOME runtime.

Reply:
Huh, I didn't see this when building. Thanks.